meet_balls_app/templates/base.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>MeetBalls</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/static/css/styles.css" rel="stylesheet" />
```

Comment:
this file doesn't exist in the repo right?

Reply:
That's correct, first one to add coustom styles will create the styles.css file, the link to it is there for future reference.